nmstatectl/nmstatectl.py

```diff
@@ -51,12 +51,19 @@ def main():
     setup_subcommand_rollback(subparsers)
```

Comment:
Awesome, it looks good. Please adjust the commit message as described in https://github.com/nmstate/nmstate/blob/master/commit-template.txt

My suggestion:

Nmstatectl: Add --version and version subcommand

No need to mention the integration tests, they are implied.
Afterwards, let's wait for @cathay4t to allow him to review the changes.

Reply:
Got it, thanks a lot.

Reply:
Please prefix the commit message with `Nmstatectl` and remove the fullstop at the end.

Reply:
This is not addressed, yet.

Reply:
Just fixed it.